binding.gyp

```diff
@@ -1,8 +1,11 @@
 {
   'targets': [
     {
+      'include_dirs':[
+        '<(module_root_dir)/node_modules/nan',
```

Comment:
Need to change binding.gyp to pull from the server node_modules directory if at all possible.